resources/views/home.blade.php

```diff
@@ -168,5 +210,18 @@ function standby(id) {
 $(document).ready(function(){
     $('.materialboxed').materialbox();
   });
+
+// for the dropdown of workshop secretaries
+var collContent = document.getElementById("workshop-administrator-content");
+var triangleSpan = document.getElementById("workshop-administrator-collapsible").firstElementChild;
+function toggleCollContent() {
+    if (collContent.style.display === "block") {
+        collContent.style.display = "none";
+        triangleSpan.textContent = "▶";
+    } else {
+        collContent.style.display = "block";
+        triangleSpan.textContent = "▼";
+    }
+}
```

Comment:
BTW we have this: https://materializecss.com/collapsible.html
but not sure if it would look nice here. You can try some options before applying my other suggestions.

You can make this cleaner with jquery, this is generated with chatgpt:
```
  var collContent = $("#workshop-administrator-content");
  var triangleSpan = $("#workshop-administrator-collapsible").find(":first-child");

  function toggleCollContent() {
    if (collContent.is(":visible")) {
      collContent.hide();
      triangleSpan.text("▶");
    } else {
      collContent.show();
      triangleSpan.text("▼");
    }
  }

  $("#workshop-administrator-collapsible").on("click", toggleCollContent);
});
```

Reply:
I've tried Materialize collapsibles; but they looked odd; that's why I chose this one. But with your jquery solution, it works nicely.